app/routes/socialEvents.tsx

```diff
@@ -1,16 +1,9 @@
 import EventCard from '~/components/common/eventCard';
-
+import { eventDetails } from '../components/common/eventCard/utils/eventDetailsObject';
 export default function socialEvents() {
   return (
     <>
-      <EventCard
-        date={30}
-        month="AUG"
-        fromTime="12AM"
-        toTime="1:30PM"
-        eventTitle="React India Conf"
-        participantCount={100}
-      />
+      <EventCard eventTitle="React India Conf" participantCount={100} eventDetails={eventDetails} />
```

Comment:
Why are values hard coded? If it is for time being add comment and remove later

Reply:
I have updated the props with exact data which we get from backend